operations/mimir-mixin/dashboards/overview-resources.libsonnet

```diff
@@ -52,13 +52,13 @@ local filename = 'mimir-overview-resources.json';
     .addRow(
       $.row('Reads')
       .addPanel(
-        $.panel('CPU') +
+        $.timeseriesPanel('CPU') +
         $.queryPanel($.resourceUtilizationQuery('cpu', $._config.instance_names.read, $._config.container_names.read), '{{%s}}' % $._config.per_instance_label),
       )
       .addPanel(
-        $.panel('Memory (workingset)') +
+        $.timeseriesPanel('Memory (workingset)') +
         $.queryPanel($.resourceUtilizationQuery('memory_working', $._config.instance_names.read, $._config.container_names.read), '{{%s}}' % $._config.per_instance_label) +
-        { yaxes: $.yaxes('bytes') },
+        { fieldConfig+: { defaults+: { unit: 'bytes' } } },
```

Comment:
nit: Overrides of the unit seems to have come up a few times, worth making a helper method in `dashboard-utils.libsonnet`?

Reply:
I was thinking if we can skip that for this PR. Maybe later we could refactor the jsonnet-lib's `timeseriesPanel()` mixin, to allow passing the "unit" argument:

```jsonnet
$.timeseriesPanel(title, unit="short")
```

For me, this fits better in the panels' JSON model